documentation/book/con-kafka-listeners.adoc

```diff
@@ -32,6 +33,60 @@ listeners:
 # ...
 ----
 
+== External listener
+
+The external listener is used to connect to a Kafka cluster from outside of an {ProductPlatformName} environment.
+{ProductName} supports three types of external listeners:
+
+* `route`
+* `loadbalancer`
+* `nodeport`
+
+Each type of external listener uses TLS encryption.
+
+.Exposing Kafka using {OpenShiftName} routes
+
+The external listener with type `route` is using {OpenShiftName} `Routes` and HAProxy to expose Kafka.
+For every Kafka broker `Pod`, a dedicated `Route` will be created.
+Additional `Route` will be created to serve as a Kafka _bootstrap_ address.
+All `Routes` use port 443.
+Kafka clients can use these `Routes` to connect to Kafka.
+
+ifdef::Kubernetes[]
+NOTE: `Routes` are available only on {OpenShiftName}. The external listener of type `route` cannot be used on {KubernetesName}.
+endif::Kubernetes[]
+
+For more information about using `Routes` to access Kafka see xref:proc-accessing-kafka-using-routes-{context}[].
+
+.Exposing Kafka using loadbalancers
+
+The external listener with type `loadbalancer` is using `Loadbalancer` type `Services` to expose Kafka.
+A loadbalancer service will be created for every Kafka broker `Pod`.
+Additional loadbalancer will be created to serve as a Kafka _bootstrap_ address.
+The loadbalancers will listen to connections on port 9094.
+For more information about using loadbalancers to access Kafka see xref:proc-accessing-kafka-using-loadbalancers-{context}[].
+
+.Exposing Kafka using node ports
+
+The external listener with type `nodeport` is using `NodePort` type `Services` to expose Kafka.
+When exposing Kafka using node ports, the Kafka clients will be connecting directly to the nodes of the {ProductPlatformName}.
+The access to the ports on the {ProductPlatformName} nodes needs to be enabled for the clients (for example in firewalls or security groups).
+Each Kafka broker `Pod` will be accessible on a different port.
+Additional `NodePort` type `Services` will be created to serve as a Kafka _bootstrap_ address.
```

Comment:
External listeners of type `nodeport` expose Kafka by using `NodePort` type `Services`. When exposing Kafka in this way, Kafka clients connect directly to the nodes of {ProductPlatformName}. You must enable access to the ports on the {ProductPlatformName} nodes for each client (for example, in firewalls or security groups). Each Kafka broker `Pod` is then accessible on a separate port. Additional `NodePort` type `Services` are created to serve as a Kafka _bootstrap_ address.